### PR TITLE
feat(content-block/item): introducing react wrappers

### PR DESCRIPTION
**Pull request by IgnacioBecerra**
### Related Ticket(s)
#8626 
#8631 

### Description
This PR adds the React wrappers for Content Block and Content Item, as well as its stories and code sandbox examples.

### Changelog

**New**

- added React wrappers for Content Block and Content Item

### CODE REVIEW

**Comment by annawen1** [comment]
@IgnacioBecerra For the `Content Block` story, when I turn on the `aside` knob I get an error

Screen Shot 2022-05-02 at 1 29 44 PM

Other than that, it looks good

File: packages/web-components/examples/codesandbox/components-react/content-item/src/index.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  padding: 2rem;
+}
+
+/* Minimum setting to use IBM Plex font */
+@font-face {
+  font-weight: 400;
+  font-family: 'IBM Plex Sans';
+  font-style: normal;
+  src: local('IBM Plex Sans'), local('IBMPlexSans'),
+    url(https://fonts.gstatic.com/s/ibmplexsans/v6/zYXgKVElMYYaJe8bpLHnCwDKhdHeEw.woff) format('woff');
```
Comment (annawen1) [inline packages/web-components/examples/codesandbox/components-react/content-item/src/index.css:25]: Ah @IgnacioBecerra can we replace the google fonts here with the our own fonts?

```

/* Minimum setting to use IBM Plex font */
@font-face {
  font-weight: 400;
  font-family: 'IBM Plex Sans';
  font-style: normal;
  src: local('IBM Plex Sans'), local('IBMPlexSans'),
    url(https://1www.s81c.com/common/carbon/plex/fonts/IBM-Plex-Sans/fonts/complete/woff/IBMPlexSans-Regular.woff) format('woff');
  font-display: auto;
}

@font-face {
  font-weight: 600;
  font-family: 'IBM Plex Sans';
  font-style: normal;
  src: local('IBM Plex Sans SemiBold'), local('IBMPlexSans-SemiBold'),
    url(https://1www.s81c.com/common/carbon/plex/fonts/IBM-Plex-Sans/fonts/complete/woff/IBMPlexSans-SemiBold.woff) format('woff');
  font-display: auto;
}
```

Comment (IgnacioBecerra) [reply packages/web-components/examples/codesandbox/components-react/content-item/src/index.css:25]: Thanks for pointing that out! I'm going to update the boilerplate script to add this 👍

**Review by annawen1 — APPROVED** [review]
LGTM! Thanks for the quick fixes!

**Comment by kennylam** [comment]
@IgnacioBecerra Looks like it's failing prettier checks.

**Pull request merged**